invenio_app_rdm/requests_ui/views/requests.py

```diff
@@ -22,6 +22,7 @@
 def requests_detail(request=None, pid_value=None):
     """Community detail page."""
     request_dict = request.to_dict()
+    default_query_config = dict(size=15)
```

Comment:
If I am not mistaken, I think the idea of having it here is to allow users to override it easily, to do so I think we should get the size from the config instead of hardcoding it here
```suggestion
    default_query_config = current_app.config[REQUESTS_SEARCH_RESULT_SIZE]
```
And add the `REQUESTS_SEARCH_RESULT_SIZE` so that it can easily be overriden by any instance. 

Feel free to change the name

Reply:
ah fair! thanks :)